.github/workflows/autopr.yml

```diff
@@ -0,0 +1,40 @@
+# Github action to run neptune-autopr on a weekly basis.
+name: Neptune Auto PR
+
+# Triggers the workflow on Fridays at 3:00-ish.
+on:
+  schedule:
+    - cron: 0 3 * * 5
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+
+    # Setting up environment with a D compiler
+    - name: Setup dlang environment
+      uses: dlang-community/setup-dlang@v1
+      with:
+        compiler: dmd-2.084.1
```

Comment:
Why such an old compiler ?

Reply:
We need to fix deprecation errors in neptune first.  Travis is still using 2.080/81.*.  This should be dealt with in another pull though.